index.html

```diff
@@ -16,37 +16,101 @@
     <div class="wrapper">
         <nav class="hamburger">
             <button type="button" class="btn-icon"></button>
+            <ul class="navbar-links">
+                <li class="actual-links">
+                    <a href="#">Hello</a>
+                </li>
+                <li class="actual-links">
+                    <a href="#">Portfolio</a>
+                </li>
+                <li class="actual-links">
+                    <a href="#">About</a>
+                </li>
+                <li class="actual-links">
+                    <a href="#">Contact</a>
+                </li>
+            </ul>
         </nav>
         <section class="container">
-            <h1 class="name">Charlie Giorgania</h1>
-            <p class="description">
-                Hello! I am a software developer! I can
-                help you build a product, feature or
-                website. Taka a look of my works.
-                If you like what you see and have a
-                project you need coded, don’t hesitate
-                and contact me.</p>
-            <button type="button" class="actual-button">
-                Start collaboration
-            </button>
-            <div class="bottom-image"></div>
+            <div class="name-container">
+                <h1 class="name">Charlie Giorgania</h1>
+                <p class="description">
+                    Hello! I am a software developer! I can
+                    help you build a product, feature or
+                    website. Taka a look of my works.
+                    If you like what you see and have a
+                    project you need coded, don’t hesitate
+                    and contact me.</p>
+                <button type="button" class="actual-button">
+                    Start collaboration
+                </button>
+            </div>
+            <div class="side-image"></div>
         </section>
+        <div class="bottom-image"></div>
         <section class="projects-section">
-            <h2 class="projects-title">Projects</h2>
```

Comment:
- [ ] On the desktop screen, your projects section is causing the page to scroll horizontally, please inspect your stylings and look for what is causing this scrolling behavior.

- [ ] On the mobile screen, your projects section and/or the next section also has the same scrolling behavior, Kindly inspect this as well.